WooCommerce/UITestsFoundation/Screens/Login/GetStartedScreen.swift

```diff
@@ -7,7 +6,7 @@ private struct ElementStringIDs {
     static let continueButton = "Get Started Email Continue Button"
 }
 
-final class GetStartedScreen: BaseScreen {
+public final class GetStartedScreen: BaseScreen {
```

Comment:
Because these types are now in a dedicated framework, we need to make them `public` to be used in the UI tests.

Notice that not all of the methods and screens had to become `public`, though. That's because only a portion of the API surface is used directly in the tests, the remainder is either unused (which I'll address as I convert them to `ScreenObject` subclasses, or only used internally.